File: TI1.3 REST_JAVA_CONUNI_GRP3/01 SERVIDOR/src/main/java/ec/edu/espe/conversionapi/enums/LengthUnit.java
package ec.edu.espe.conversionapi.enums;

/**
 * Unidades de longitud
 */
public enum LengthUnit {
    METERS("Metros"),
    KILOMETERS("Kilómetros"),
    CENTIMETERS("Centímetros"),
    MILLIMETERS("Milímetros"),
    MILES("Millas"),
    YARDS("Yardas"),
    FEET("Pies"),
    INCHES("Pulgadas");

    private final String displayName;

    LengthUnit(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: TI1.4 REST_DOTNET_CONUNI_GRP3/02 CLIMOV/convrtfull/app/src/main/java/ec/edu/espe/conv_rtfull/data/model/LengthUnit.kt
package ec.edu.espe.conv_rtfull.data.model

/**
 * Unidades de longitud
 */
enum class LengthUnit(val displayName: String, val code: Int) {
    MILLIMETERS("Milímetros", 3),
    CENTIMETERS("Centímetros", 2),
    METERS("Metros", 0),
    KILOMETERS("Kilómetros", 1),
    INCHES("Pulgadas", 7),
    FEET("Pies", 6)
}
